.github/workflows/main.yml

```diff
@@ -10,9 +10,12 @@ on:
 
 env:
   IMAGE: responses-dashboard
-  REGISTRY_HOSTNAME: eu.gcr.io
+  REGISTRY_HOSTNAME: europe-west2-docker.pkg.dev
   HOST: ${{ secrets.GOOGLE_PROJECT_ID }}
-  RELEASE_HOST: ${{ secrets.RELEASE_PROJECT_ID }}
+  GAR_REPOSITORY: ${{ secrets.GAR_REPOSITORY }}
+#  Temp override
+#  RELEASE_HOST: ${{ secrets.RELEASE_PROJECT_ID }}
+  RELEASE_HOST: ons-ci-rmrasbs
```

Comment:
This is only a temp variable which will be removed once RELEASE_PROJECT_ID has been updated to the correct release host.